tests/userController.test.ts

```
@@ -0,0 +1,79 @@
+    });
+
+    expect(mockRes.status).toHaveBeenCalledWith(204);
+    expect(mockRes.send).toHaveBeenCalled();
+  });
+
+  it("should fail if no userId is provided", async () => {
+    mockReq.apiKey = undefined;
+    await deleteMe(mockReq as any, mockRes as any, mockNext);
+
+    expect(mockNext).toHaveBeenCalled();
+    expect(mockNext.mock.calls[0][0].message).toBe("User-scoped API key required");
```

ESLint

[error] Replace `"User-scoped·API·key·required"` with `⏎······"User-scoped·API·key·required",⏎····`

(prettier/prettier)